src/test_util.rs

```diff
@@ -14,6 +14,25 @@ pub fn test_lint(
   let diagnostics = linter
     .lint(filename.to_string(), source_code.to_string(), rules)
     .expect("Failed to lint");
-  let serialized_diagnostics = serde_json::to_value(diagnostics).unwrap();
+
+  // TODO(@disizali) refactor this to a better approach.
+  // it's a temporary solution for ignoring line_src field.
```

Comment:
we could write `"line_src"` field on every single expected_diagnostic on each test, but it's redundant IMHO; 
because we do line number assertion anyway.

Reply:
@disizali don't worry about testing utility - I'll intend to rewrite it shortly. Any hacks in here are fine 👍